test/utils/creds.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/spyzhov/ajson"
+)
+
+// Credentials returns the credentials from the creds.json file.
+func Credentials() (*ajson.Node, error) {
+	fileName := "creds.json"
```

Comment:
Can this be a parameter to the function instead? There will be more than one creds file for the different tests, within the Hubspot tests, can pass in `hubspot-creds.json` as the parameter, and within the Salesforce tests, can pass in `salesforce-creds.json`, etc.